lib/compiler/src/beam_disasm.erl

```diff
@@ -1308,8 +1308,8 @@ resolve_inst({bs_match,[{Fail,Ctx,{z,1},{u,_},Args}]},_,_,_) ->
 %% OTP 27.
 %%
 
-resolve_inst({executable_line,[Index]},_,_,_) ->
-    {line,resolve_arg(Index)};
+resolve_inst({executable_line,[Location,Index]},_,_,_) ->
+    {line,resolve_arg(Location),resolve_arg(Index)};
```

Comment:
Should this be translated as `{executable_line,Location,Index}` instead? I was about to open a separate pull to suggest this since it's useful to know the difference between line and executable_line but now I think it makes even more sense since it's no longer the same as `{line,Index}`

```suggestion
    {executable_line,resolve_arg(Location),resolve_arg(Index)};
```

Reply:
Thanks for noticing this. Fixed!